pandas/core/computation/pytables.py

```diff
@@ -563,7 +563,7 @@ def __init__(
         self._visitor = None
 
         # capture the environment if needed
-        local_dict: DeepChainMap[Any, Any] = DeepChainMap()
+        local_dict: Optional[DeepChainMap[Any, Any]] = None
```

Comment:
did this change any tests?

Reply:
Oh, I was waiting for the test and didn't see your message. Yes, turned out this part of code relied on `bool(DeepChainMap())` being equal to `False`. This fixed all the issues, waiting for pre-commit now